test/e2e/autoscale.go

```diff
@@ -161,7 +161,8 @@ func generateTraffic(
 			totalRequests++
 			if res.Code != http.StatusOK {
 				ctx.logf("Status = %d, want: 200", res.Code)
-				ctx.logf("URL: %s Duration: %v Error: %s Body:\n%s", res.URL, res.Latency, res.Error, string(res.Body))
+				ctx.logf("URL: %s Start: %v End: %v, Duration: %v Error: %s Body:\n%s",
```

Comment:
nit picking, but none of the others are comma separated, and time.Time will be long. Maybe use %q for res.End() or something?

Reply:
Then probably even time.Format? :)

Reply:
Done.